src/synopsys-action/tools-parameter.ts

```diff
@@ -218,13 +218,14 @@ export class SynopsysToolsParameter {
     return command
   }
 
-  private setGithubData(): GithubData | undefined {
+  private setGithubData(isCommentFlow: boolean): GithubData | undefined {
```

Comment:
why are we taking it as method param, can't we check inside as input. ?

Reply:
if we use inputs then again parsing and check for both coverity and blackduck

Reply:
updated